docs/howto/assets/eks/scripts/install-worker.sh

```diff
@@ -0,0 +1,191 @@
+
+MACHINE=$(uname -m)
+if [ "$MACHINE" == "x86_64" ]; then
+    ARCH="amd64"
+else
+    echo "Unknown machine architecture '$MACHINE'" >&2
+    exit 1
+fi
+
+################################################################################
+### Packages ###################################################################
+################################################################################
+
+# Update the OS to begin with to catch up to the latest packages.
+sudo yum update -y
+
+# Install necessary packages
+sudo yum install -y \
+    aws-cfn-bootstrap \
+    awscli \
+    chrony \
+    conntrack \
+    curl \
+    jq \
+    ec2-instance-connect \
+    nfs-utils \
+    socat \
+    unzip \
+    wget \
+    git \
+    openssl-devel
+
+sudo yum group install -y "Development Tools"
+
+# Remove the ec2-net-utils package, if it's installed. This package interferes with the route setup on the instance.
+if yum list installed | grep ec2-net-utils; then sudo yum remove ec2-net-utils -y -q; fi
+
+################################################################################
+### Time #######################################################################
+################################################################################
+
+# Make sure Amazon Time Sync Service starts on boot.
+sudo chkconfig chronyd on
+
+# Make sure that chronyd syncs RTC clock to the kernel.
+cat <<EOF | sudo tee -a /etc/chrony.conf
+# This directive enables kernel synchronisation (every 11 minutes) of the
+# real-time clock. Note that it can’t be used along with the 'rtcfile' directive.
+rtcsync
+EOF
+
+# If current clocksource is xen, switch to tsc
+if grep --quiet xen /sys/devices/system/clocksource/clocksource0/current_clocksource &&
+  grep --quiet tsc /sys/devices/system/clocksource/clocksource0/available_clocksource; then
+    echo "tsc" | sudo tee /sys/devices/system/clocksource/clocksource0/current_clocksource
+else
+    echo "tsc as a clock source is not applicable, skipping."
+fi
+
+################################################################################
+### kubectl ####################################################################
+################################################################################
+
+curl -o kubectl https://amazon-eks.s3.us-west-2.amazonaws.com/1.15.10/2020-02-22/bin/linux/amd64/kubectl
+sudo chmod +x kubectl
+sudo chown root:root kubectl
+sudo mv kubectl /usr/local/bin 
+
+################################################################################
+### Krustlet ###################################################################
+################################################################################
+
+# Install a Rust toolchain
+curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y
+export PATH=$PATH:$HOME/.cargo/bin
+
+# Build krustlet to link against the system libssl
+# TODO: make the krustlet to build (wasi or wascc) configurable
```

Comment:
Just checking, is this to build it against openssl 1.0 instead of 1.1?

Reply:
That's correct.  Would an additional comment help clarify that?